src/BProject/code first/Model/Order.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace code_first.Model
+{
+    class Order
+    {
+    
+        
+        public Order()
+        {
+            this.OrderItems = new HashSet<OrderItem>();
+        }
+
+        public int ID { get; set; }
+        public Nullable<int> CustomerID { get; set; }
+        public Nullable<decimal> TotalAmount { get; set; }
+        public Nullable<bool> IsPayed { get; set; }
+        public Nullable<int> PaymentTypeID { get; set; }
+        public Nullable<int> Status { get; set; }
```

Comment:
```suggestion
        public Nullable<int> StatusID { get; set; }
```

Będzie trzeba w bazie zmienić, później Status na StatusID